config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
Maybe here you can use 
'''resources :users, only: [:new]'''
I just think it's nicer and you can easly add methods  ;)

Reply:
i will :) thanks